kotskinds/apis/kots/v1beta1/installation_types.go

```diff
@@ -29,7 +27,7 @@ type InstallationSpec struct {
 	ChannelName   string                  `json:"channelName,omitempty"`
 	VersionLabel  string                  `json:"versionLabel,omitempty"`
 	ReleaseNotes  string                  `json:"releaseNotes,omitempty"`
-	ReleasedAt    *time.Time              `json:"releasedAt,omitempty"`
+	ReleasedAt    *metav1.Time            `json:"releasedAt,omitempty"`
```

Comment:
@codelingo when any types in this directory are modified, the zz_generated.deepcopy.go file must be updated as well.